game/scripts/vscripts/Sections/Chapter5/SectionOpening.ts

```diff
@@ -80,75 +78,70 @@ function onStart(complete: () => void) {
+                    tg.panCameraExponential(shared.runeSpawnsLocations.direBotBountyPos, shared.runeSpawnsLocations.direAncientsBountyPos, panCameraBountiesAlpha),
                     tg.setCameraTarget((ctx) => ctx[CustomEntityKeys.DireAncientsBountyRune]),
                     tg.immediate((ctx) => {
-                        RemoveFOWViewer(DotaTeam.GOODGUYS, ctx[CustomEntityKeys.DireBotBountyFOWViewer])
                         ctx[CustomEntityKeys.DireAncientsBountyFOWViewer] = AddFOWViewer(DotaTeam.GOODGUYS, shared.runeSpawnsLocations.direAncientsBountyPos, 800, visionRevealDuration, false)
                     }),
                     tg.wait(visionRevealDuration),
-
-                    tg.immediate(ctx => {
-                        RemoveFOWViewer(DotaTeam.GOODGUYS, ctx[CustomEntityKeys.DireAncientsBountyFOWViewer])
-                    }),
                 ]),
             ]),
 
             // Return camera to player
             tg.fork([
                 tg.audioDialog(LocalizationKey.Script_5_Opening_4, LocalizationKey.Script_5_Opening_4, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
-                tg.panCameraLinear(_ => getPlayerCameraLocation(), _ => playerHero.GetAbsOrigin(), 1),
+                tg.panCameraExponential(_ => getPlayerCameraLocation(), _ => playerHero.GetAbsOrigin(), panCameraBountiesAlpha),
             ]),
 
             tg.audioDialog(LocalizationKey.Script_5_Opening_5, LocalizationKey.Script_5_Opening_5, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
-            tg.audioDialog(LocalizationKey.Script_5_Opening_6, LocalizationKey.Script_5_Opening_6, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
-
-            tg.fork([
+            tg.forkAny([
                 tg.seq([
-                    tg.panCamera(_ => getPlayerCameraLocation(), shared.runeSpawnsLocations.topPowerUpRunePos, slowerCameraSpeedFunc),
-                    tg.wait(1),
-                    tg.setCameraTarget(undefined),
-                    tg.immediate(_ => centerCameraOnHero()),
-                    tg.immediate((ctx) => {
-                        canPlayerIssueOrders = true
-                        goalMoveToRune.start()
-                        if (IsValidEntity(ctx[CustomEntityKeys.TopPowerRune])) {
-                            ctx[CustomEntityKeys.TopPowerRune].Destroy()
-                            ctx[CustomEntityKeys.TopPowerRune] = undefined
-                        }
-                    }),
+                    tg.audioDialog(LocalizationKey.Script_5_Opening_6, LocalizationKey.Script_5_Opening_6, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+                    tg.neverComplete()
                 ]),
-                tg.goToLocation(shared.runeSpawnsLocations.topPowerUpRunePos.__add(Vector(-300, 100, 0)), [GetGroundPosition(Vector(-3250, 1600), undefined)]),
+                tg.fork([
+                    tg.seq([
```

Comment:
can remove this seq; also the fork above then

Reply:
Need the seq since it's wrapped in `forkAny`? I can remove the fork though and move `goToLocation` inside the seq